### ISSUE TITLE
Crash to Desktop

### ISSUE THREAD

**Issue by anathemamask** [SkyrimTogether/issues-game#96]
## Description

Operating ST on two PCs, same mod setup, differing hardware, both windows 10. One does not have this issue, the other does.
PC hardware report:
[Hardware.txt](https://github.com/SkyrimTogether/issues-game/files/2798185/Hardware.txt)

Related formIDs|
---------------|
[Flames - 12FCD](http://en.uesp.net/wiki/Skyrim:Flames)|
[Nazeem - 1A6A4](http://en.uesp.net/wiki/Skyrim:Nazeem)|

## Steps to reproduce

How to reproduce this issue:

Crashes ALSO occur from main menu.
1. Open SkyrimSE through Harbour.
2. Be invited, join party, launch party. (crash to desktop)
3. Attempt to: save, pick flowers, (crash to desktop)

## Reproduction rate

How many times can you reproduce it?
Erratically. Happens 5-20 minutes apart.
## Expected result

Should not crash to desktop.

## Your environment

* Game edition (choose on which edition do you have problems):
        * The Elder Scrolls V: Skyrim Special Edition

## Evidence (optional)
Dump files located at https://drive.google.com/open?id=158LR7lvH1KUdBI0sZhB6DOKJtBwEF2ZX

Hardware report is included in above hardware.txt

**Comment by KBDCDN** [comment]
This crash to desktop happened to my party of five as well, with some members lasting longer than others. We have a hunch that fast traveling can cause it (multiple crashes rarely), but it sometimes happens with no apparent reason. It happened inconsistently over a five hour period, with my peak break between crashes lasting nearly 30 minutes.

I've noticed that if the game crashes, it can sometimes affect your save, causing you to crash every time you move too far from your starting position. For example, I crashed near the Whiterun gate and continued to do so if I loaded in and walked toward the gate itself. Another party member suggested loading your previous save, which has worked in every instance this happened, even if the save was made moments before the last. 

I should also add that saving works for me even when connected to the server, with other players in the party reporting that it DOES cause them to freeze or crash entirely. 


Attached are my hardware specifications and dmp files:
- https://github.com/SkyrimTogether/issues-game/files/2799470/Hardware.txt
- https://drive.google.com/drive/folders/1m4yqoBsapy9LfHV1JLjtVVWuYNo9vvme?usp=sharing

**Comment by anathemamask** [comment]
Update: After intensive testing tonight I seem to have narrowed down the cause of this in my case. I can support KBDCDN's thoughts about corrupted saves, replicated by loading a savegame that had been experiencing a lot of CTDs. 


Steps to resolve:
Disabled all addons except - SKSE64 2.0.12, Alternate Start, Unofficial Skyrim SE Patch.
Created new characters, ruling out corrupt saves caused by initial launch attempts. 
Played for over an hour connected to one another with 0 ctd, completed a few small quests and attempted Bleak Falls Barrow (dragonstone missing, no big deal, known issue.) 


TLDR of below: Potential repeatable CTD if players are moving when they hit the quicksave button. SKYUI does not appear to be the cause of any negative mod interactions.  Graphical mods seem confirmed as not having an effect on ST. The Immersive Encounters mod, which had previously been enabled as defacto, is likely the culprit of our CTD as its the only addon we did not enable, it heavily modifies NPC interaction.
---
After verifying no issues from that mod setup, we tried again after enabling SkyUI.
One CTD occurred after 13 minutes, but it was when I had cast a Summoning spell and the creature entered into combat. Replicated that issue by trying it again. Same CTD. Stopped casting summon spells. SKYUI was ruled not the issue. 

During this period we entered a simple cave, and noted a semi-repeatable CTD, if players are moving while quicksave is pressed a CTD can occur, seems about one in every 5 times. Perhaps related to position updates during saves? A good safe practice is everyone to take a brief moment and stop moving until the quicksave finishes processing. 

We signed out after completing the cave and enabled more mods, including: Morehud, Skyrim Revamped Combat Overhaul, SPERG, TES Master Sound Track, Barenziah Quest Markers, Achieve That, Animallica, Blowing in the wind, smim, cloaks of skyrim, point the way, WICO, Dr. Bandolier, RS Children overhaul. immersive children, ks hairdos, realistic water II, rebalanced level lists, srg enhanced trees, enhanced blood, embers hd. Legacy of the dragonborn, cloaks of skyrim.  A significant amount of additional addons besides. 

The only addons we did not enable from our list were: Immersive Encounters, Realistic AI detection. 

One of those two is presumed the cause of the CTD.

No further crashes to desktop after a total of 5 hours of play in 30+ minute spurts.

Unrelated: Legacy of the dragonborn is broken by the lack of container sync, and has really unpredictable behaviour.

**Comment by F1r3w477** [comment]
My friend and I play with a majority of graphics overhaul mods, he has the issue, while I do not. We use none of the above-listed mods that might be the issue.

**Comment by Pokang** [comment]
@F1r3w477 I understand that but as you can see in Ana's comment it is definitely caused by mods which I can't assure you the stability of, along with Skyrim Together. 
Except if your friend is on Windows 7, which we gave a fix for here : https://www.reddit.com/r/SkyrimTogether/comments/ajxkwr/all_known_fixes_for_issues_with_skyrim_together/

**Comment by F1r3w477** [comment]
@Pokang I'm aware, I was just backing up the post by saying I use all different mods, and still have the issue. (Which means it doesn't matter what mods people have, it just happens.)

**Comment by Pokang** [comment]
@F1r3w477 If you can narrow down the issue and report to us what mod causes crashes I'd appreciate it a lot. Thank you!

**Comment by storm1er** [comment]
The last update is actually removing every player (non npc) while executing a quicksave, so I think we can close this issue.

Also, the "new" version of this could be #172?
